src/Organizations/OrganizationEdit/OrganizationEdit.js

```diff
@@ -42,17 +49,32 @@ export const OrganizationEdit = ({ match, history, location, mutator }) => {
     isLoading: isBankingInformationLoading,
   } = useOrganizationBankingInformation(organizationId);
 
-  useEffect(
-    () => {
-      mutator.editOrganizationOrg.GET()
-        .then(organizationsResponse => {
-          setOrganization(organizationsResponse);
-        })
-        .finally(() => setIsOrganizationLoading(false));
-    },
+  const fetchOrganization = useCallback(() => {
```

Comment:
Please get rid off this and use `useOrganization` instead https://github.com/folio-org/stripes-acq-components/blob/master/lib/hooks/useOrganization/useOrganization.js#L10